backend/Makefile

```diff
@@ -84,6 +84,9 @@ db/test_migration:
 	$(MAKE) db/dump_schema > /tmp/after
 	diff /tmp/{before,after} # No news is good news.
 
+SSH_SERVER_ALIVE_INTERVAL_IN_SECONDS?=60
+SSH_SERVER_ALIVE_COUNT_MAX?=60
 db/tunnel:
 	$(eval endpoint=$(shell aws rds describe-db-cluster-endpoints --db-cluster-identifier corpora-${DEPLOYMENT_STAGE}-corpora-api | jq '.DBClusterEndpoints[] | select(.EndpointType | contains("WRITER")) | .Endpoint'))
-	ssh -L 5432:${endpoint}:5432 bastion.${DEPLOYMENT_STAGE}.single-cell.czi.technology
+	ssh -N -o ServerAliveInterval=${SSH_SERVER_ALIVE_INTERVAL_IN_SECONDS} -o ServerAliveCountMax=${SSH_SERVER_ALIVE_COUNT_MAX} \
```

Comment:
also `-f -T`? for backgrounding the ssh tunnel and no pseudo-terminal allocation

I think we also used `-M` for `"master" mode for connection sharing`, and tracking active tunnels via files, but maybe just have that as a TODO?